packages/teams-js/test/private/conversations.spec.ts

```diff
@@ -114,7 +112,11 @@ describe('conversations', () => {
     it('closeConversation should throw error if conversation capability is not supported in runtime config', async () => {
       await utils.initializeWithContext(FrameContexts.content);
```

Comment:
Looks like I missed a place where we need `expect.assertions`

Reply:
Okay I'll add it every where